Include/internal/pycore_pystate.h

```diff
@@ -20,15 +21,8 @@ _Py_IsMainThread(void)
     return (thread == _PyRuntime.main_thread);
 }
 
-
-static inline int
-_Py_IsMainInterpreter(PyInterpreterState *interp)
-{
-    /* Use directly _PyRuntime rather than tstate->interp->runtime, since
-       this function is used in performance critical code path (ceval) */
-    return (interp == _PyRuntime.interpreters.main);
-}
-
+extern int
+_Py_IsMainInterpreter(PyInterpreterState *interp);
```

Comment:
Why change this from static inline?

Reply:
That may have been necessary for linkage at some point, but it looks like it should be fine now. I'll change it back.